ref-exchange/src/account_deposit.rs

```diff
@@ -5,29 +5,100 @@ use std::convert::TryInto;
 
 use near_sdk::borsh::{self, BorshDeserialize, BorshSerialize};
 use near_sdk::json_types::{ValidAccountId, U128};
-use near_sdk::{assert_one_yocto, env, near_bindgen, AccountId, Balance, PromiseResult};
+use near_sdk::{
+    assert_one_yocto, env, near_bindgen, AccountId, Balance, PromiseResult, StorageUsage,
+};
 
 use crate::utils::{ext_fungible_token, ext_self, GAS_FOR_FT_TRANSFER};
 use crate::*;
 
-const MAX_ACCOUNT_LENGTH: u128 = 64;
-const MIN_ACCOUNT_DEPOSIT_LENGTH: u128 = MAX_ACCOUNT_LENGTH + 16 + 4;
+const U128_STORAGE: StorageUsage = 16;
+/// bytes length of u64 values
+const U64_STORAGE: StorageUsage = 8;
+/// bytes length of u32 values. Used in length operations
+const U32_STORAGE: StorageUsage = 4;
+/// max length of account id * 8 (1 byte)
+const ACC_ID_STORAGE: StorageUsage = 64 * 8;
```

Comment:
hm, why is it by 8? it's just 64 byte max.

Reply:
ah, ok!